src/components/blend-model.js

```diff
@@ -0,0 +1,40 @@
+var registerComponent = require('../core/component').registerComponent;
+var THREE = require('../lib/three');
+
+var loader = new THREE.ColladaLoader();
+loader.options.convertUpAxis = true;
+
+module.exports.Component = registerComponent('blend-model', {
+  schema: {
+    type: 'src'
+  },
+
+  init: function () {
+    this.model = null;
+  },
+
+  update: function () {
+    var self = this;
+    var el = this.el;
+    var src = this.data;
+
+    if (!src) { return; }
+
+    this.remove();
+    this.model = new THREE.BlendCharacter();
+
+    this.model.load(src, function () {
+      el.setObject3D('mesh', self.model);
+      el.emit('model-loaded', {format: 'blend', model: self.model});
+      self.model.castShadow = true;
+      self.model.receiveShadow = true;
+      self.model.material.shading = THREE.FlatShading;

```

Comment:
the same as before, probably we should expose `cast/receiveShadows` another way instead of enabling them by default in this kind of components